my-app/src/routes/api/members/+server.ts

```diff
@@ -6,3 +6,9 @@ export const GET: RequestHandler = async ({ url, cookies }) => {
 	const body = await groupService.listAllMembers(+groupId, cookies);
 	return new Response(JSON.stringify(body));
 };
+
+export const DELETE: RequestHandler = async ({ url, cookies }) => {
+	const groupId = url.searchParams.get('groupId') || 0;
+	const body = await groupService.leaveGroup(+groupId, cookies);
```

Comment:
Ojo que acá groupId puede ser NaN
```suggestion
	const groupId = +url.searchParams.get('groupId') || 0;
	const body = await groupService.leaveGroup(groupId, cookies);
```

Reply:
🤯🤯🤯